Source/WebCore/platform/graphics/cocoa/FontCacheCoreText.cpp

```diff
@@ -668,6 +669,23 @@ static void autoActivateFont(const String& name, CGFloat size)
 
 std::unique_ptr<FontPlatformData> FontCache::createFontPlatformData(const FontDescription& fontDescription, const AtomString& family, const FontCreationContext& fontCreationContext, OptionSet<FontLookupOptions> options)
 {
+    if (auto& fontMap = userInstalledFontMap()) {
+        auto fontKey = family.string().convertToASCIILowercase();
+        auto fontURL = fontMap->getOptional(fontKey);
+        if (fontURL) {
+            RELEASE_LOG(Fonts, "FontCache user installed fonts contains family %s with fontURL %s", family.string().utf8().data(), fontURL->string().utf8().data());
+            RetainPtr<CFURLRef> cfURL = fontURL->createCFURL();
+
+            CFErrorRef error = nullptr;
+            if (!CTFontManagerRegisterFontsForURL(cfURL.get(), kCTFontManagerScopeProcess, &error)) {
+                RetainPtr<CFStringRef> descriptionCF = adoptCF(CFErrorCopyDescription(error));
```

Comment:
Do we really need `<CFStringRef>`?

Reply:
Good point, that should not be needed, will fix!